cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md

```diff
@@ -178,17 +178,15 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
 > "I want to (re)validate the 2010 datasets within my setup…"
 
 - … for the MinimumBias, Commissioning, Mu or MuMonitor datasets:
-    - Go to 2010 Validation Utilities,
+    - Go to [Validation software](http://opendata-dev.web.cern.ch/search?page=1&size=20&q=&type=Software&subtype=Validation&experiment=CMS), FIXME Year in the search
     - choose and execute the corresponding Validation code.
 
 - … for the Multijet dataset (see also [here](http://hep.caltech.edu/cms/opendata/)):
-    - Go to 2010 CMS Tools,
-    - choose and execute `Razor filter and analyzer for SUSY searches`.
+    - choose and execute [Razor filter and analyzer for SUSY searches](http://opendata-dev.web.cern.ch/record/553).
 
 - … for the Electron (or Mu) dataset:
-    - Go to 2010 CMS Tools,
-    - choose and execute `Software to preprocess the CMS 2010 Muon and Electron datasets for the two-lepton/four-lepton analysis example of CMS open data`, then
-    - choose `Two-lepton/four-lepton analysis example of CMS 2010 open data` and compare PAT-tuples from the previous to those linked therein, or execute it on your new PAT tuples.
+    - choose and execute [Software to preprocess the CMS 2010 Muon and Electron datasets for the two-lepton/four-lepton analysis example of CMS open data](http://opendata-dev.web.cern.ch/record/200), then
```

Comment:
`- choose and execute ["Software to preprocess the CMS 2010 Muon and Electron datasets for the two-lepton/four-lepton analysis example of CMS open data"]`